src/openrct2/scripting/ScNetwork.hpp

```diff
@@ -0,0 +1,377 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2018 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef __ENABLE_SCRIPTING__
+
+#    include "../actions/NetworkModifyGroupAction.hpp"
+#    include "../actions/PlayerKickAction.hpp"
+#    include "../actions/PlayerSetGroupAction.hpp"
+#    include "../network/NetworkAction.h"
+#    include "../network/network.h"
+#    include "Duktape.hpp"
+
+namespace OpenRCT2::Scripting
+{
+    class ScPlayerGroup
```

Comment:
Class is using snake_case should be CamelCase

Probably should be named `ScriptPlayerGroup` (file `ScriptNetwork`?)